sway-lsp/src/lib.rs

```diff
@@ -9,8 +11,16 @@ mod server;
 pub mod test_utils;
 pub mod utils;
 use server::Backend;
+use tracing::metadata::LevelFilter;
 
 pub async fn start() {
+    let tracing_options = TracingSubscriberOptions {
+        log_level: Some(LevelFilter::DEBUG), // TODO: Set this based on IDE config
```

Comment:
sounds good, we can have a chat next week and I can talk you though how to go about setting something like this up.